package.json

```diff
@@ -78,11 +78,13 @@
     "react": "17.0.2",
     "react-native": "0.66.3",
     "react-native-bootsplash": "^3.2.4",
+    "react-native-custom-tabs": "^0.1.7",
     "react-native-dash": "^0.0.11",
     "react-native-device-info": "^8.3.1",
     "react-native-geolocation-service": "^5.2.0",
     "react-native-gesture-handler": "^1.10.3",
     "react-native-get-random-values": "^1.7.0",
+    "react-native-inappbrowser-reborn": "^3.7.0",
```

Comment:
Just out of curiosity, why is this needed? don't we have existing webview implementations that does the same or is this different?

Reply:
We do have Webview implementations in the App, however, Vipps documentation asks the apps to use Custom tabs and not Webviews. https://github.com/vippsas/vipps-login-api/blob/master/vipps-login-api.md#using-vipps-login-in-native-applications 
So this basically opens Chrome Custom Tabs for Android & Safari for iOS.